README.md

```diff
@@ -1,21 +1,30 @@
 # wavefront-lambda-go
 
-[![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go)
-[![Go Report Card](https://goreportcard.com/badge/github.com/wavefrontHQ/wavefront-lambda-go)](https://goreportcard.com/report/github.com/wavefrontHQ/wavefront-lambda-go)
+[![travis build status](https://travis-ci.com/retgits/wavefront-lambda-go.svg?branch=master)](https://travis-ci.com/retgits/wavefront-lambda-go)
+[![Go Report Card](https://goreportcard.com/badge/github.com/retgits/wavefront-lambda-go)](https://goreportcard.com/report/github.com/retgits/wavefront-lambda-go)
+[![GoDoc reference](https://godoc.org/github.com/retgits/wavefront-lambda-go?status.svg)](https://godoc.org/github.com/retgits/wavefront-lambda-go)
+[![Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](./LICENSE)
 
-A Go wrapper for AWS Lambda so you can monitor everything from your [Wavefront](https://wavefront.com) dashboard
+> A Go wrapper for AWS Lambda so you can monitor everything from your [Wavefront](https://wavefront.com) dashboard.
+
+The package `wavefront-lambda-go` provides a Go wrapper for AWS Lambda function so you can monitor the three pillars of observability from your [Wavefront](https://wavefront.com) dashboard.
```

Comment:
regarding: three pillars of observability 
This wrapper only provides metrics.

Reply:
You're right 😄 I'll make a note for myself to also add the option to send spans and trace events. In the meanwhile, I've removed the line.